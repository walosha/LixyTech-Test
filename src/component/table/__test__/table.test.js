import React from "react";
import ReactDom from "react-dom";
import { render } from "@testing-library/react";
import TestRenderer from "react-test-renderer";

import Table from "../table";

it("Render Table Successfully ", () => {
  const div = document.createElement("div");
  ReactDom.render(<Table />, div);
});

it("Render Table Successfully ", () => {
  const tree = TestRenderer.create(<Table />).toJSON();
  expect(tree).toMatchSnapshot();
});
