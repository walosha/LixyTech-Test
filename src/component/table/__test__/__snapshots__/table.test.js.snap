// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Table Successfully  1`] = `
<table
  className="table table-striped"
>
  <thead>
    <tr>
      <th>
        #
      </th>
    </tr>
  </thead>
  <tbody />
</table>
`;
